pandas/tests/arrays/string_/test_string.py

```diff
@@ -171,3 +171,19 @@ def test_arrow_array():
     arr = pa.array(data)
     expected = pa.array(list(data), type=pa.string(), from_pandas=True)
     assert arr.equals(expected)
+
+
+@td.skip_if_no("pyarrow", min_version="0.15.1.dev")
```

Comment:
is this still accurate (the version number)?

Reply:
Yes, the released version will be 1.0, but using 0.15.1.dev here so the tests run with arrow master